### ISSUE TITLE
Android application not loading on older devices

### ISSUE THREAD

**Issue by mikeller** [betaflight/betaflight-configurator#2581]
Reproduction:
- start the android app.

Expected behaviour:
- the loading screen (wasp in front of honeycomb) shows;
- it switches to the landing screen.

Actual behaviour:
- the landing screen stays on.

System information:
- hardware: Nexus 7 tablet;
- os: android 5.1.1;
- application version: betaflight-configurator_10.8.0-debug-5cc28aae

Error:
`Failed to load module script: Expected a JavaScript module script but the server responded with a MIME type of "". Strict MIME type checking is enforced for module scripts per HTML spec.`

![image](https://user-images.githubusercontent.com/4742747/131047758-6e15358a-e52d-4c05-a740-b6c1aa27b158.png)

**Comment by mikeller** [comment]
@Asizon: #2136 was merged, and it is still broken, so there seems to be another problem.

**Comment by WalcoFPV** [comment]
Reproduced on Samsung S7 edge running Android 9 and Chrome webview 74.0.3729.136
@Asizon I think, it's not a Webview related issue. After updating Chrome (version 92.0.4515.159), I have still the issue.

**Comment by mikeller** [comment]
@WalcoFPV: I suspect so too - it looks like the version of webview available on older versions of android is more restrictive as to not inferring `text/javascript` as the type for files loaded with `type="module"` that have no type.
So this seems to have been broken when we started moving parts of configurator to modules in 10.80.
According to https://github.com/apache/cordova-android/issues/1142#issuecomment-776322077 there does not seem to be a straightforward fix for this, and the only workaround seems to be the use of extra tooling. I have tried to fix this in #2582 by adding the ionic webview - it seems to fix the issue on my Nexus 7, but not sure if this is a good fix.
